NRLMMD-GEOIPS/geoips
# update pytest ini and integrate base tests into pytest integration tests

* [x] NO REQUIRED ***unit tests***
* [x] NO REQUIRED ***updates to other repos***

# Related Issues
fixes NRLMMD-GEOIPS/geoips#834

# Testing Instructions
- Read the added docs, and try to follow them. If they don't make sense we should change them!

# Summary
- Removed shell script integration tests
- Replaced them with pytest tests, separated base and full sets
- Added new pytest markers `base` and `full` to allow running of one or the other 
- Added documentation
